### PR TITLE
[17.0] [MIG] mgmtsystem_hazard

### PR DESCRIPTION
**Pull request by ThinkopenDev**
Straightforward migration from 16.0

### CODE REVIEW

**Comment by mymage** [comment]
@ThinkopenDev please re-push so runboat will go green.

File: pandoc-3.1.12.1-1-amd64.deb
Comment (mymage) [inline pandoc-3.1.12.1-1-amd64.deb:1]: I think that this sliped inside the commit but must be deleted.

**Review by mymage — CHANGES_REQUESTED** [review]

File: pandoc-3.1.12.1-1-amd64.deb
Comment (ThinkopenDev) [reply pandoc-3.1.12.1-1-amd64.deb:1]: Sorry, this PR was closed on accident.
Meanwhile, we've pushed a new commit to remove the *.deb files.
I can't find option to re-open this PR, should we create a new one or is it OK for now?

Comment (mymage) [reply pandoc-3.1.12.1-1-amd64.deb:1]: If I remember well you have the possibility to reopen a PR you have closed, but I may be wrong.
I see the new PR, I will put comment there.

**Comment by mymage** [comment]
Hi @ThinkopenDev, the repo was fixed (see #585).
Then I think you have to re-push this after a rebase. Thanks!

**Review by dreispt — CHANGES_REQUESTED** [review]
I'm sorry, this migration attempt got the history wrong, and rebase doesn't work.
It needs to be re-done using the recipe linked in the migration Issue.

**Pull request closed without merging**